Source/WebCore/dom/AbortSignal.h

```diff
@@ -84,7 +86,8 @@ class AbortSignal final : public RefCounted<AbortSignal>, public EventTarget, pr
     void eventListenersDidChange() final;
     
     bool m_aborted { false };
-    Vector<Algorithm> m_algorithms;
+    Vector<std::pair<uint32_t, Algorithm>> m_algorithms;
+    uint32_t m_algorithmIdentifier { 0 };
```

Comment:
Can this be an ObjectIdentifier of some sort?

Reply:
I thought about this, the issue is that the algorithm identifier is stored by built-in JS code which only knows about integers, not ObjectIdentifier.